flink/v1.19/flink/src/test/java/org/apache/iceberg/flink/source/TestIcebergSourceFailoverWithWatermarkExtractor.java

```diff
@@ -48,10 +52,17 @@ public class TestIcebergSourceFailoverWithWatermarkExtractor extends TestIceberg
 
   @Override
   protected IcebergSource.Builder<RowData> sourceBuilder() {
-    return IcebergSource.<RowData>builder()
+    Configuration config = new Configuration();
+    config.setInteger(FlinkConfigOptions.SOURCE_READER_FETCH_BATCH_RECORD_COUNT, 128);
+    return IcebergSource.forRowData()
         .tableLoader(sourceTableResource.tableLoader())
         .watermarkColumn("ts")
-        .project(TestFixtures.TS_SCHEMA);
+        .project(TestFixtures.TS_SCHEMA)
+        // Prevent combining splits
+        .set(
```

Comment:
ah. we missed the bundling earlier.

Reply:
Yeah, but the error was not with this one 😄.
I have just find it when copied the other configs to here.